src/index.coffee

```diff
@@ -68,15 +67,15 @@ getRequests = (root) ->
     models = []
 
     # get all indexes defined in models into an array
-    for file in fs.readdirSync modelPath
+    for file in fs.readdirSync modelsPath
         try
-            model = require modelPath + file
+            model = require modelsPath + file
```

Comment:
This code is wrong. It should be done via the `path` module and its `join` function.